libraries/AP_HAL_ChibiOS/hwdef/CubeOrangePlus-dv/defaults.parm

```diff
@@ -1,630 +1,467 @@
+AVOID_BEHAVE,0
+AVOID_DIST_MAX,5
+AVOID_ENABLE,3
+AVOID_MARGIN,2
+
+BATT_AMP_PERVLT,50
+BATT_ARM_VOLT,42
+BATT_CAPACITY,22000
+BATT_CRT_VOLT,38
+BATT_FS_CRT_ACT,1
+BATT_FS_LOW_ACT,2
+BATT_FS_VOLTSRC,0
+BATT_LOW_TIMER,10
+BATT_LOW_VOLT,40
+BATT_MONITOR,4
+BATT_VOLT_MULT,15.31
+
+BRD_BOOT_DELAY,1000
+BRD_HEAT_LOWMGN,20
+BRD_HEAT_TARG,45
+BRD_SAFETY_DEFLT,0
+BRD_TYPE,3
+BRD_VBUS_MIN,4.3
+
+CAN_D1_PROTOCOL,1
+CAN_D1_PROTOCOL2,0
+CAN_D1_UC_NODE,10
+CAN_D1_UC_NTF_RT,5
+CAN_D1_UC_OPTION,2
+CAN_D1_UC_POOL,16384
+CAN_D1_UC_SRV_RT,1
+CAN_D2_PROTOCOL,1
+CAN_D2_PROTOCOL2,0
+CAN_D2_UC_NODE,10
+CAN_D2_UC_NTF_RT,20
+CAN_D2_UC_POOL,16384
+CAN_D2_UC_SRV_RT,50
+CAN_P1_BITRATE,1000000
+CAN_P1_DRIVER,1
+CAN_P1_FDBITRATE,8
+CAN_P2_BITRATE,1000000
+CAN_P2_DRIVER,2
+CAN_P2_FDBITRATE,8
+CAN_SLCAN_CPORT,1
+CAN_SLCAN_SDELAY,1
+CAN_SLCAN_SERNUM,-1
+CAN_SLCAN_TIMOUT,2
+
+COMPASS_AUTO_ROT,2
+COMPASS_AUTODEC,1
+COMPASS_CAL_FIT,16
+COMPASS_DISBLMSK,32
+COMPASS_ENABLE,1
+COMPASS_FLTR_RNG,0
+COMPASS_LEARN,0
+COMPASS_MOTCT,0
+COMPASS_USE,1
+COMPASS_USE2,0
+COMPASS_USE3,0
+
+DID_BARO_ACC,0
+DID_CANDRIVER,0
+DID_ENABLE,1
+DID_MAVPORT,1
+DID_OPTIONS,0
+
+DISARM_DELAY,2
+
+EK3_ABIAS_P_NSE,0.003
+EK3_ALT_M_NSE,3
+EK3_BCN_DELAY,50
+EK3_BCN_I_GTE,500
+EK3_BCN_M_NSE,1
+EK3_BETA_MASK,0
+EK3_CHECK_SCALE,100
+EK3_DRAG_BCOEF_X,109.1
+EK3_DRAG_BCOEF_Y,87
+EK3_ENABLE,1
+EK3_IMU_MASK,7
+EK3_OGN_HGT_MASK,2
+EK3_RNG_USE_HGT,-1
+EK3_RNG_USE_SPD,2
+EK3_SRC_OPTIONS,0
+EK3_SRC1_POSXY,3
+EK3_SRC1_POSZ,3
+EK3_SRC1_VELXY,3
+EK3_SRC1_VELZ,3
+EK3_SRC1_YAW,3
+EK3_SRC2_POSXY,3
+EK3_SRC2_POSZ,3
+EK3_SRC2_VELXY,3
+EK3_SRC2_VELZ,3
+EK3_SRC2_YAW,1
+EK3_SRC3_POSXY,0
+EK3_SRC3_POSZ,1
+EK3_SRC3_VELXY,0
+EK3_SRC3_VELZ,0
+EK3_SRC3_YAW,1
+
+FENCE_ACTION,1
+FENCE_ALT_MAX,120
+FENCE_ENABLE,1
+FENCE_MARGIN,1
+FENCE_RADIUS,500
+FENCE_TYPE,3
+
+FLTMODE1,2
+FLTMODE2,2
+FLTMODE3,5
+FLTMODE4,4
+FLTMODE5,6
+FLTMODE6,9
+
+FRAME_CLASS,1
+FRAME_TYPE,1
+
+FS_CRASH_CHECK,1
+FS_DR_ENABLE,2
+FS_DR_TIMEOUT,30
+FS_EKF_ACTION,2
+FS_EKF_FILT,5
+FS_EKF_THRESH,0.8
+FS_GCS_ENABLE,1
+FS_GCS_TIMEOUT,5
+FS_OPTIONS,24
+FS_THR_ENABLE,1
+FS_THR_VALUE,975
+FS_VIBE_ENABLE,1
+
+GND_EFFECT_COMP,1
+
+GPS_AUTO_CONFIG,1
+GPS_AUTO_SWITCH,1
+GPS_BLEND_MASK,5
+GPS_HDOP_GOOD,140
+GPS_INJECT_TO,127
+GPS_MB1_OFS_X,0
+GPS_MB1_OFS_Y,-0.74
+GPS_MB1_OFS_Z,0
+GPS_MB1_TYPE,1
+GPS_MB2_TYPE,0
+GPS_POS1_X,0.36
+GPS_POS1_Y,0.37
+GPS_POS1_Z,-0.05
+GPS_POS2_X,0.36
+GPS_POS2_Y,-0.37
+GPS_POS2_Z,-0.05
+GPS_PRIMARY,0
+GPS_SAVE_CFG,2
+GPS_TYPE,17
+GPS_TYPE2,18
+
+INITIAL_MODE,5
+
+INS_ACC_BODYFIX,2
+INS_ACCEL_FILTER,10
+INS_ENABLE_MASK,127
+INS_FAST_SAMPLE,7
+INS_GYR_CAL,1
+INS_GYRO_FILTER,60
+INS_GYRO_RATE,2
+INS_HNTC2_ENABLE,0
+INS_HNTCH_ATT,25
+INS_HNTCH_BW,9
+INS_HNTCH_ENABLE,1
+INS_HNTCH_FREQ,30
+INS_HNTCH_HMNCS,15
+INS_HNTCH_MODE,3
+INS_HNTCH_OPTS,38
+INS_HNTCH_REF,1
+INS_LOG_BAT_CNT,2048
+INS_LOG_BAT_MASK,5
+INS_LOG_BAT_OPT,5
+
+INS_USE,1
+INS_USE2,1
+INS_USE3,1
+
+LAND_ALT_LOW,500
+LAND_REPOSITION,1
+LAND_SPEED,40
+
+LOG_BACKEND_TYPE,1
+LOG_BITMASK,145405

```

Comment:
Camera, Motors and Fast IMU logs are disabled by this bitmask. If it is desired? I'd add Motors logs back.

Reply:
Not sure why you mention Motors are disabled by this bitmask. It was set like this from the beginning I didn't change anything. BTW motors are not disabled in this setting

Reply:
So, the default is everything on, but we disabled; 

- Medium Attitude
- Optical Flow
- Camera
- Motors
- Fast IMU
- Video Stabilization
- Fast harmonic notch logging

I mark this as solved.

Reply:
Motors are not disabled from logging!

Reply:
So, there seems to be a discrepancy between the documentation and the setup in Mission Planner: 
https://ardupilot.org/copter/docs/parameters-Copter-stable-V4.5.1.html#log-bitmask-log-bitmask
![image](https://github.com/user-attachments/assets/a59fe83b-561c-4211-9e40-40ee0227743c)

Reply:
+ 1 reason to do NOT use MP.